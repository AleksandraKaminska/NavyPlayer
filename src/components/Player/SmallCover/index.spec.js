import React from 'react'
import { BrowserRouter as Router } from 'react-router-dom'
import { SmallCover } from './index'
import renderer from 'react-test-renderer'

describe('SmallCover', () => {
  it('should render self', () => {
    const rendered = renderer.create(
      <Router>
        <SmallCover />
      </Router>
    )
    expect(rendered.toJSON()).toMatchSnapshot()
  })
})
