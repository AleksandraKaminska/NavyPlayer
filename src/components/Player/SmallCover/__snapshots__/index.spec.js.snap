// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallCover should render self 1`] = `
<a
  href="/"
  onClick={[Function]}
>
  <div
    className="small"
  >
    <img
      alt="cover"
      className="smallCover"
    />
    <div
      className="smallTitle"
    >
      <p />
      <p />
    </div>
  </div>
</a>
`;
